modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/portlet/action/ImportObjectFolderMVCActionCommand.java

```diff
@@ -73,8 +74,15 @@ protected void doProcessAction(
 
 			JSONObject jsonObject = null;
 
-			if (exception instanceof
-					ObjectFolderItemObjectDefinitionIdException) {
+			if (exception instanceof ObjectFolderNameException) {
+				Class<?> clazz = exception.getClass();
+
+				jsonObject = JSONUtil.put(
+					"type",
+					"ObjectFolderNameException." + clazz.getSimpleName());
+			}
+			else if (exception instanceof
+						ObjectFolderItemObjectDefinitionIdException) {
```

Comment:
@gabrielwas, could you review or sign another reviewer for these BE modifications?

Furthermore, I was unable to have a clear opinion about the tests carried out. These tests are passing routine and failed this PR, but these failures have already appeared in previous runs. In other words, they can be flaky.

When trying to run on my local master, the tests fail for another reason.

Reply:
Hey @igor-franca! I have some questions regarding this. Could you reach me latter? Thx

Reply:
Talked to @igor-franca! LGTM :) Thx!